Polyfills/Canvas/Source/Context.cpp

```diff
@@ -598,17 +613,42 @@ namespace Babylon::Polyfills::Internal
                 const auto width = m_canvas->GetWidth();
                 const auto height = m_canvas->GetHeight();
 
+                nanovg_filterstack::InitBgfx(); // HACK: should this even be a static on nanovg_filterstack?
```

Comment:
This init will happen each time canvas is dirty. So basically each frame with experience.js sample code. This code is called at Plugin initialization : https://github.com/Pheo/BabylonNative/blob/b64e17a2b1e7bb7b6d5a74e7d52824f9ed04520e/Polyfills/Canvas/Source/Context.cpp#L42
bgfx uniforms will only need to be created once.